.pre-commit-config.yaml

```diff
@@ -0,0 +1,90 @@
+exclude: "^(docs|.devcontainer|.venv|.vscode|templates|.*tests.*snapshots.*).*|.*[.]pyi"
+default_language_version:
+  ruby: 2.7.2
```

Comment:
Felipe, maybe here should be python 3.10?

Reply:
The default language does not need to be specified so I deleted the line.